### PR TITLE
Added contributing guide lines

### PR DESCRIPTION
**Pull request by eveyonline**

### CODE REVIEW

**Review by qroques — APPROVED** [review]

**Review by NicolasNSSM — APPROVED** [review]
This PR clearly will make the futur of this project cleaner and greater, thank you!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+## Code of Conduct
+
+This project and everyone participating in it is governed by the following
+[Code of Conduct](https://github.com/KnpLabs/bad-jokes/blob/master/CODE_OF_CONDUCT.md).
+By participating, you are expected to uphold this code.
+
+## Reporting a bug
+
+#### Before submitting a bug
```
Comment (qroques) [inline CONTRIBUTING.md:19]: It seems it has a jump of title level 

```suggestion
### Before submitting a bug

```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+- Verify that you are using the latest server-side-renderer version;
+- Double-check the [documentation](https://github.com/KnpLabs/bad-jokes/blob/master/README.md)
+to see if you're not misusing the library;
+- Check if the issue has already been reported. If it has, and the issue is still
+open, add a comment to the existing issue instead of opening a new one.
+
+#### How to submit a (good) bug report
```
Comment (qroques) [inline CONTRIBUTING.md:26]:
```suggestion
### How to submit a (good) bug report

```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+When submitting a pull request please be sure to follow the same rules described
+above in [Reporting a bug](#reporting-a-bug) and [Suggesting enhancements](#suggesting-enhancements)
+sections depending on the nature of your change.
+
+> Before starting to work on a large change please open an issue to ask the
+maintainers if they are fine with it (no one likes to work for nothing!).
+
+1. Fork the repository
+2. Once the repository has been forked clone it locally
+```
```
Comment (qroques) [inline CONTRIBUTING.md:58]:
```suggestion
```shell
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+git clone git@github.com:USERNAME/bad-jokes.git
+```
+3. Create a new branch
+```
```
Comment (qroques) [inline CONTRIBUTING.md:62]:
```suggestion
```shell
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+git checkout -b BRANCH_NAME master
+```
+4. Code!!!
+5. Add/Update tests (if needed)
+6. Update documentation (if needed)
+7. Run the tests and make sure that they are passing
+```
```
Comment (qroques) [inline CONTRIBUTING.md:69]:
```suggestion
```shell
```

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+make test
+```
+8. Fix code style issues
+```
```
Comment (qroques) [inline CONTRIBUTING.md:73]:
```suggestion
```shell
```

**Review by qroques — CHANGES_REQUESTED** [review]

**Comment by qroques** [comment]
:ah:

**Pull request merged**